Apps/APN/src/screens/Settings.js

```diff
@@ -65,6 +65,7 @@ const Settings = ({ navigation, route }) => {
             {...props}
             style={styles.backButton}
             labelVisible={Platform.OS === 'ios'}
+            accessibilityLabel="Navigate up"
```

Comment:
I don't think we have any accessibility value such as "Navigate up" 🤔

Reply:
Yes you are correct, we don't have one. "Navigate up" is added to default back button by React Native and I didn't find any easy way to change it. So I matched this with default button to keep it consistent for react app, will update tests to use "Navigate up" instead of "Back Button" for react native apps.

Reply:
Okay, but I am not sure if changing it would help Jason test the app using appium. 🤔

Reply:
I added a workaround to fix this for [Flutter previously](https://github.com/customerio/mobile-apps-automation-tests/blob/main/src/driver/element-assistant.ts#L41-L44), the same approach can help us override this for React Native Apps as well 🤓